app/utilities/schema.py

```diff
@@ -92,8 +92,8 @@ def _schema_exists(language_code, schema_name):
     )
 
 
-def get_allowed_languages(schema_name, launch_language):
```

Comment:
No need for nested list and then just:
```python
def get_allowed_languages(launch_language):
    if launch_language in LANGUAGES_MAP:
        return language_combination
    return [DEFAULT_LANGUAGE_CODE]
```

Reply:
Tried this but needed to revert it because removing the for loop broke it because its being called with multiple languages

Reply:
Yeah that was meant for flattened list, then it would work, but since you removed this change now it doesn't matter anymore.